tests/conftest.py

```diff
@@ -14,3 +16,9 @@ def mock_env() -> Iterator[os._Environ[str]]:
     """
     with mock.patch.dict(os.environ, clear=True):
         yield os.environ
+
+
+@pytest.fixture()
+def _load_yaml_file_clear_cache() -> Iterator[None]:
```

Comment:
```suggestion
def clear_load_yaml_file_cache() -> Iterator[None]:
```

Reply:
It's a [convention](https://docs.astral.sh/ruff/rules/pytest-missing-fixture-name-underscore/)

Reply:
Removed the leading underscore and removed the `PT` rules from `ruff`.

[reason](https://github.com/astral-sh/ruff/issues/8796)